network/udp/net.go

```diff
@@ -23,7 +23,13 @@ type Network struct {
 
 // NewNetwork creates Nework baked by udp protocol
 func NewNetwork(addr string, enc network.Encoding) (*Network, error) {
-	udpAddr, err := net.ResolveUDPAddr("udp4", addr)
+	_, port, err := net.SplitHostPort(addr)
+	if err != nil {
+		return nil, err
+	}
+	newAddr := net.JoinHostPort("0.0.0.0", port)
```

Comment:
why do you have to do this ? It sounds a bit "rough" to force listening on all interfaces - there might be unintended consequences. If it's a good reason (and I'm sure it is xD ), then at least documenting it would be nice so it's public information.

Reply:
this = listening on 0.0.0.0

Reply:
well only this was working for me, what would you recommend?

Reply:
What did not work, in what context (udp in aws?)  ? One should be able to listen on a given IP of one interface - the address given in parameter ?

Reply:
Context udp on aws, when I passed the addr (parameter) it didn't work after changing to 0.0.0.0 it started working. Just tested it again to confirm it.